aws/internal/service/glue/id.go

```diff
@@ -30,3 +32,9 @@ func stringifyAwsGluePartition(partValues *schema.Set) string {
 
 	return vals
 }
+
+func CreateAwsGlueRegistryID(id string) *glue.RegistryId {
```

Comment:
Nit. My understanding is that `Create` and `Parse` ID functions are meant to work on strings only, so this functionality can just be moved into `finder.RegistryByID`.